docs/api/text/freqdist.rst

```diff
@@ -7,7 +7,7 @@ A method for visualizing the frequency of tokens within and across corpora is fr
 
 .. code:: python
 
-    from yellowbrick.text.freqdist import FreqDistVisualizer
+    from yellowbrick.text import FreqDistVisualizer
```

Comment:
I was not certain about this change; however, when I tested the code I was able to import FreqDistVisualizer using both::

from yellowbrick.text.freqdist import FreqDistVisualizer
from yellowbrick.text import FreqDistVisualizer

Since they both worked, I decided to err on the side of succinctness. However, I wanted to check with you to make sure that I wasn't missing something.